### ISSUE TITLE
CSS not being injected

### ISSUE THREAD

**Issue by ankitjainOfficial** [yeoman/generator-webapp_DEPRECATED#579]
The generator is running fine, only thing is that the CSS is not being injected and have to do a manual refresh. 

Though, css is being generated. 

```
Completed in 2.435s at Sat Aug 01 2015 13:13:32 GMT+0530 (IST) - Waiting...
>> File "app/styles/main.scss" changed.
Running "sass:server" (sass) task

Running "postcss:dist" (postcss) task
>> 1 processed stylesheet created.

Done, without errors.


Execution Time (2015-08-01 07:45:44 UTC)
loading tasks          482ms  ▇▇▇▇▇▇ 15%
loading grunt-sass     130ms  ▇▇ 4%
sass:server            809ms  ▇▇▇▇▇▇▇▇▇▇▇ 26%
loading grunt-postcss  769ms  ▇▇▇▇▇▇▇▇▇▇ 24%
postcss:dist           948ms  ▇▇▇▇▇▇▇▇▇▇▇▇ 30%
Total 3.1s

Completed in 3.999s at Sat Aug 01 2015 13:15:47 GMT+0530 (IST) - Waiting...
```

**Comment by silvenon** [comment]
Which version are you running? Do you have JavaScript enabled?

**Comment by ankitjainOfficial** [comment]
generator-webapp 1.0.1 and yes, javascript is enabled.

**Comment by tulsadome** [comment]
Im having this exact same problem. I spent a few hours trying to edit the gruntfile with no luck and I ended up switching to gulp-webapp which works perfectly with all the same code.

**Comment by Caesor** [comment]
hi @ankitjainOfficial , have you fixed this error?  I have the same problem.

**Comment by ankitjainOfficial** [comment]
No. I don't have much knowledge with grunt. I generally use gulp, but this
is one of the projects where I needed to use this generator.

**Comment by philwolstenholme** [comment]
I'm also seeing this with version 1.0.1, for a project freshly scaffolded on 14/08/2015.

**Comment by silvenon** [comment]
Probably related to BrowserSync/grunt-browser-sync#117.

**Comment by silvenon** [comment]
It seems that grunt-browser-sync [isn't necessary](http://www.shakyshane.com/javascript/nodejs/browser-sync/2014/08/24/browser-sync-plus-grunt/). I'll implement this solution one of these days, in the meantime you guys can try it if you're in an urgent situation.

**Comment by hersonls** [comment]
Hey guys, I had the same problem and what I notice

**Comment by hersonls** [comment]
Hey guys, I had the same problem here. What I noticed was that, when using a dotfile directory ".tmp", it ignores the files. I modified the directory ".tmp" to just "tmp" and it worked perfectly.

I spent a few hours looking for a solution to this problem, to keep the ".tmp" working. Maybe you can help me with this. I saw something about dot[0] in grunt Files configuration, but no success with in my tries.

[0] - http://gruntjs.com/configuring-tasks#files

**Comment by silvenon** [comment]
I heard from multiple people that removing `.` from `.tmp` worked for them. I wonder why, it doesn't seem to make any sense :confused:

**Comment by silvenon** [comment]
The fix is released in [v1.1.0](https://github.com/yeoman/generator-webapp/releases/tag/v1.1.0).
